Shipr-Helm/templates/redis-pvc.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ Values.redis.name }}-pv
+spec:
+  capacity:
+    storage: {{ Values.redis.volume.storage }}
+  accessModes:
+    - {{ Values.redis.volume.accessModes }}
+  hostPath:
+    path: {{ Values.redis.volume.hostPath }}
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ Values.redis.name }}-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: {{ Values.redis.storage }}
```

Comment:
Any reason for not using a storage class?

Reply:
Yes storage class is better option here